app/views/users/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_with(model: user, local: true) do |form| %>
+  <% if user.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+      <ul>
+      <% user.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
- [ ] use a helper method to render the if/end statement in this part of your code there should be no if/end statements within views if you have several lines you can use a partial to render the markup